src/vsphere_cpi/spec/unit/cloud/vsphere/attach_tag_to_vm_spec.rb

```diff
@@ -7,6 +7,7 @@ module TaggingTag
     describe AttachTagToVm, fake_logger: true do
       subject(:api_client) { VSphereAutomation::ApiClient.new }
       subject(:tagging_tag) { TaggingTag::AttachTagToVm.new(api_client) }
+      let(:vm_mob) { instance_double('VimSdk::Vim::VirtualMachine') }
 
       describe '#retrieve_category_id' do
         let(:result_1) do
```

Comment:
Is this useful?

Reply:
We could use `anything` instead of `vm_mob`. Tried to make the test a little more fail-proof by making sure only `vm_mob` object is received.